.github/workflows/test.yml

```diff
@@ -234,70 +232,10 @@ jobs:
           target: ${{ matrix.os.name }}
           github-token: "${{ secrets.GITHUB_TOKEN }}"
 
+      # TODO: ensure integration tests runs with both GO_TAG: go and GO_TAG: rust
```

Comment:
This won't matter soon (hopefully)

Reply:
how soon? Should I find a way to run both in this PR or block this PR until we don't need it anymore? Is https://github.com/vercel/turbo/pull/4181 the thing that makes it so we don't need both?

Reply:
Yeah once we flip the rust switch (hopefully) we won't switch it back and we can start deleting Go code. I'd wait until the switch PR lands and then we can merge this